Controller/ElasticsearchController.php

```diff
@@ -647,14 +643,6 @@ public function exportAction(Request $request, JobService $jobService, ContentTy
     }
 
     /**
-     * @param Request $request
-     * @return RedirectResponse|Response
-     * @throws Throwable
```

Comment:
Keep throws. And return type as it is not defined in the function call

Reply:
Rollbacked